src/App.tsx

```diff
@@ -1,39 +1,62 @@
-import React from 'react';
+import { Component } from 'react';
 import './App.scss';
+import { Clock } from './components/Clock/Clock';
```

Comment:
looks like you didn't push all files
![Screenshot from 2023-07-07 12-38-46](https://github.com/mate-academy/react_clock/assets/33029864/20f4ec2e-3561-4468-a2d8-f8495879632c)
![Screenshot from 2023-07-07 12-38-36](https://github.com/mate-academy/react_clock/assets/33029864/697e2204-97b1-443f-96ac-ec86668501e4)